README.md

```diff
@@ -287,11 +287,29 @@ To do this, just follow these simple steps:
 
 To destroy the provisioned infrastructure, just run `rake terraform:destroy`
 
+
 ## Installer tool
 
 ```
 ruby -e "$(curl -fsSL https://raw.githubusercontent.com/openware/opendax/master/bin/install)"
 ```
 
+
+## Using opendax deployment for local development
+
+Modify `templates/config/gateway/envoy.yaml.erb` file by next way:
+
+1. Set cors allow_origin as "*"
+2. Setup all allow_methods what you need. Example: `allow_methods: "PUT, GET, POST, DELETE, PATCH"`
+3. Setup expose_headers. Example: `expose_headers:  "total, page, x-csrf-token"`
+4. Add `allow_credentials: true` to your cors configuration
```

Comment:
Please replace this with an Envoy config snippet with each of these points as comments